contracts/fixed_price.scilla

```diff
@@ -1106,6 +1123,26 @@ procedure Cancel(
   end
 end
 
+procedure BatchCancelOrder_(order: CancelOrderParam)  
```

Comment:
@razill can you rename to `DoBatchCancelOrder` to be similar to how other procedures are named?

Reply:
Thanks @inamirandaz, will update